src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,12 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     *
+     * @return true if data is mutated by command
+     */
+    public boolean isMutating(){
+        return false;
```

Comment:
Having a default implementation for the `isMutating` method may not be intuitive for a class like `Command` as it is abstract in context. Why return false and not true?

Perhaps a better implementation would be make this method abstract and force all classes extended from `Command` to implement their own method definition.